exporter-helper.php

```diff
@@ -118,7 +118,13 @@ function get_campaign_attachments( $post_ids ) {
 		$sql, // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
 		array_merge( [ $wpdb->posts ], $post_ids )
 	);
-	$attachment_ids = $wpdb->get_results( $prepared_sql, OBJECT_K ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+	$results        = $wpdb->get_results( $prepared_sql ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
```

Comment:
Maybe you can instead use a function that ensures the result set comtains an integer and not an object? Then you wouldn't need the `array_map` below.

Reply:
@Inwerpsel - can we move with current changes for this release and work on improvements in coming sprint?